src/components/AI/Vertex.js

```diff
@@ -100,36 +100,36 @@ class Vertex {
   }
     
   /**
-    * Gets this vertex's location
-    * @return this vertex's location (PVector)
-    */
+     * Gets this vertex's location
+     * @return this vertex's location (PVector)
+     */
   getLoc() {
```

Comment:
Can you fix up the javadoc spacing here? They should match up with the first '*' column. There's a couple other places in the file, too.